mappings/net/minecraft/client/render/item/HeldItemRenderer.mapping

```diff
@@ -1,6 +1,29 @@
 CLASS net/minecraft/class_759 net/minecraft/client/render/item/HeldItemRenderer
 	FIELD field_21807 MAP_BACKGROUND Lnet/minecraft/class_1921;
 	FIELD field_21808 MAP_BACKGROUND_CHECKERBOARD Lnet/minecraft/class_1921;
+	FIELD field_32689 ARM_HOLDING_ITEM_FIRST_Z_ANGLE_MULTIPLIER F
+	FIELD field_32699 ARM_HOLDING_ITEM_TRANSLATE_X F
+	FIELD field_32700 ARM_HOLDING_ITEM_X_ANGLE_MULTIPLIER I
+	FIELD field_32701 ARM_HOLDING_ITEM_THIRD_Y_ANGLE_MULTIPLIER I
+	FIELD field_32702 ARM_HOLDING_ITEM_SECOND_Z_ANGLE_MULTIPLIER I
+	FIELD field_32712 ARM_X_ANGLE_MULTIPLIER F
+	FIELD field_32713 ARM_Y_ANGLE_MULTIPLIER F
+	FIELD field_32714 ARM_Z_ANGLE_MULTIPLIER F
+	FIELD field_32715 ARM_TRANSLATE_X F
+	FIELD field_32716 ARM_TRANSLATE_Y F
+	FIELD field_32717 ARM_TRANSLATE_Z F
+	FIELD field_32719 FIRST_PERSON_MAP_FIRST_SCALE F
+	FIELD field_32720 FIRST_PERSON_MAP_TRANSLATE_X F
+	FIELD field_32721 FIRST_PERSON_MAP_TRANSLATE_Y F
+	FIELD field_32722 FIRST_PERSON_MAP_TRANSLATE_Z F
+	FIELD field_32723 FIRST_PERSON_MAP_SECOND_SCALE F
+	FIELD field_32739 EQUIP_OFFSET_TRANSLATE_X F
+	FIELD field_32740 EQUIP_OFFSET_TRANSLATE_Y F
+	FIELD field_32741 EQUIP_OFFSET_TRANSLATE_Z F
+	FIELD field_32746 EAT_OR_DRINK_X_ANGLE_MULTIPLIER F
+	FIELD field_32747 EAT_OR_DRINK_Y_ANGLE_MULTIPLIER F
+	FIELD field_32748 EAT_OR_DRINK_Z_ANGLE_MULTIPLIER F
+	FIELD field_32758 ARM_HOLDING_ITEM_SECOND_Y_ANGLE_MULTIPLIER F
```

Comment:
I'm confused on some of these fields. What is `first`, `second`, and `third` here?

Reply:
It means it's used in the first/second/third call of `matrices.multiply()`  in the method